cmd2/rl_utils.py

```diff
@@ -113,11 +114,19 @@ def pyreadline_remove_history_item(pos: int) -> None:
 elif 'gnureadline' in sys.modules or 'readline' in sys.modules:
     # We don't support libedit
     if 'libedit' not in readline.__doc__:
-        rl_type = RlType.GNU
-
         # Load the readline lib so we can access members of it
-        import ctypes
-        readline_lib = ctypes.CDLL(readline.__file__)
+        try:
+            import ctypes
```

Comment:
Why would an import of ctypes fail?

Reply:
ctypes is a module.  It doesnt need to exist, and is one of the first to be disabled in embedded Python, especially on new arch/platforms where the entire gcc toolchain isnt available/stable yet.  In a completely static build of Python with all needed modules built in, there is no need for ctypes, and it adds bloat.